### PR TITLE
fix russian translation

### PR DESCRIPTION
**Pull request by kloun**

### CODE REVIEW

**Comment by kloun** [comment]
bot, add author to whitelist

**Comment by kloun** [comment]
bot, test pull request

**Comment by matthiasclasen** [comment]
@rh-atomic-bot r+

**Comment by matthiasclasen** [comment]
Looks like something is wrong here:

> /usr/bin/msgmerge --update  --lang=ru ru.po flatpak.pot
ru.po:4260:11: syntax error
ru.po:4260: keyword "s" unknown

File: po/ru.po [hunk 1]
```diff
@@ -4253,12 +4252,12 @@ msgstr "%s требует более новую версию flatpak (%s)"
...
 #: common/flatpak-utils.c:5710
 msgid "Empty string is not a number"
-msgstr ""
+msgstr "Пустая строка не является числом"
 
 #: common/flatpak-utils.c:5736
 #, c-format
 msgid "“%s” is not an unsigned number"
-msgstr ""
+msgstr ""%s" не является положительным числом"
```
Comment (mwleeds) [inline po/ru.po]: This should be `“%s”` rather than `"%s"`

**Comment by mwleeds** [comment]
@rh-atomic-bot r+ 781e1b8

**Pull request closed without merging**